src/connectors/scenes/ChangeMiningFeeSendConfirmationConnector.ui.js

```diff
@@ -18,7 +18,7 @@ export const mapStateToProps = (state: State, ownProps: ChangeMiningFeeOwnProps)
   if (_.has(wallet, 'currencyInfo.defaultSettings.customFeeSettings')) {
     customFeeSettings = wallet.currencyInfo.defaultSettings.customFeeSettings
   }
-  const hideCustomFeeOption = !!Constants.getSpecialCurrencyInfo(wallet.currencyInfo.currencyCode).noChangeMiningFee
+  const hideCustomFeeOption = !!Constants.getSpecialCurrencyInfo(wallet.currencyInfo.currencyCode).noCustomMiningFee
```

Comment:
This breaks EOS which has neither a custom mining fee nor ANY ability to change mining fee. Can you put in separate logic for each.